src/vcpkg/postbuildlint.cpp

```diff
@@ -240,6 +240,28 @@ namespace vcpkg::PostBuildLint
         return LintStatus::SUCCESS;
     }
 
+    static LintStatus check_for_usage_forgot_install(const Filesystem& fs,
+                                                     const VcpkgPaths& paths,
+                                                     const Path& package_dir,
+                                                     const PackageSpec& spec)
+    {
+        static const std::string STANDARD_INSTALL_USAGE =
+            R"###(file(INSTALL "${CMAKE_CURRENT_LIST_DIR}/usage" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}"))###";
+
+        auto port_path = paths.root / "ports" / spec.name();
```

Comment:
That is not true for overlay ports and custom registries or overwritten versions.

Reply:
Is there any exist way to get the port folder?

Reply:
You can pass this variable further down:
https://github.com/microsoft/vcpkg-tool/blob/f39cfece245139aed1dfb4d7d71b9adf61c0018a/src/vcpkg/postbuildlint.cpp#L1210

Reply:
Got it